setup.py

```diff
@@ -0,0 +1,58 @@
+from os.path import isdir, join
+from platform import system
+
+from setuptools import Extension, find_packages, setup
+from setuptools.command.build import build
+from wheel.bdist_wheel import bdist_wheel
+
+
+class Build(build):
+    def run(self):
+        if isdir("queries"):
+            dest = join(self.build_lib, "tree_sitter_swift", "queries")
+            self.copy_tree("queries", dest)
+        super().run()
+
+
+class BdistWheel(bdist_wheel):
+    def get_tag(self):
+        python, abi, platform = super().get_tag()
+        if python.startswith("cp"):
+            python, abi = "cp38", "abi3"
+        return python, abi, platform
+
+
+setup(
+    packages=find_packages("bindings/python"),
+    package_dir={"": "bindings/python"},
+    package_data={
+        "tree_sitter_swift": ["*.pyi", "py.typed"],
+        "tree_sitter_swift.queries": ["*.scm"],
+    },
+    ext_package="tree_sitter_swift",
+    ext_modules=[
+        Extension(
+            name="_binding",
+            sources=[
+                "bindings/python/tree_sitter_swift/binding.c",
+                "src/parser.c",
+                # NOTE: if your language uses an external scanner, add it here.
+                "src/parser.c",
```

Comment:
I think this is meant to be `scanner.c`.

Reply:
Weird, I didn’t think I touched this.